src/storage/mvcc/reader/scanner/forward.rs

```diff
@@ -504,15 +504,15 @@ impl<S: Snapshot> ScanPolicy<S> for LatestEntryPolicy {
                     break Some(TxnEntry::Commit {
                         default: entry_default,
                         write: entry_write,
-                        old_value: OldValue::None,
+                        old_value: OldValue::Unspecified,
```

Comment:
Could you elaborate why change to `Unspecified`?

Reply:
Fixed by updating `tests::EntryBuilder`.